misc/python/materialize/buildkite_insights/step_durations/analysis.py

```diff
@@ -176,6 +190,8 @@ def main(
     parser.add_argument("--build-step-key", default=None, type=str)
     parser.add_argument("--no-fetch", action="store_true")
     parser.add_argument("--max-fetches", default=5, type=int)
+    parser.add_argument("--branch", default="main", type=str)
+    parser.add_argument("--build-state", default=None, type=str)
```

Comment:
What are the possible build states? the argument parser can be used to validate that a valid string has been provided.

Reply:
✅ Adjusted.